File: Codeforces/Golang/800-1000/469A - I Wanna Be the Guy/main.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)

	levels := make(map[int]struct{})

	var p int
	fmt.Scan(&p)

	for i := 0; i < p; i++ {
		var l int
		fmt.Scan(&l)
		levels[l] = struct{}{}
	}

	var q int
	fmt.Scan(&q)
	for i := 0; i < q; i++ {
		var l int
		fmt.Scan(&l)
		levels[l] = struct{}{}
	}

	if len(levels) == n {
		fmt.Println("I become the guy.")
	} else {
		fmt.Println("Oh, my keyboard!")
	}
}

File: Codeforces/Rust/800-1000/469A - I Wanna Be the Guy/src/main.rs
use std::collections::HashSet;
use std::io;

fn main() {
    let mut input = String::new();
    let _ = io::stdin().read_line(&mut input).unwrap();
    let n: usize = input.trim().parse().unwrap();

    let mut levels = HashSet::new();

    for _ in 0..2 {
        let mut line = String::new();
        let _ = io::stdin().read_line(&mut line).unwrap();
        let mut parts = line.split_whitespace();

        parts.next();

        for num_str in parts {
            let level: i32 = num_str.parse().unwrap();
            levels.insert(level);
        }
    }

    if levels.len() == n {
        println!("I become the guy.")
    } else {
        println!("Oh, my keyboard!")
    }
}
